javascript-if/main.js

```diff
@@ -0,0 +1,97 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro
+ */
+
+function isUnderFive(number) {
+  let result;
+  var underFive = number < 5;
+  if ((number < 5)) {
+    result = underFive;
+  } else {
+    result = underFive;
+  }
+  return result;
+}
+
```

Comment:
This seems a little redundant.
All you need to do is return a boolean if the number is under five
```
function isUnderFive(n) {
  if (n < 5) {
    return true
  } else {
    return false
  }
}